src/main/java/com/google/sps/data/TextAnalyser.java

```diff
@@ -210,45 +208,86 @@ public AnalyzeSyntaxResponse analyseSyntaxText() throws IOException {
     }
   }
 
-  public Set<String> getAdjectives() throws IOException {
+  // Get the combination of verb-adverb
+  public Set<String> getSyntax() throws IOException {
     Set<String> adjectives = new LinkedHashSet<String>();
+    String verb = "";
+    boolean add = false;
 
     for (Token token : analyseSyntaxText().getTokensList()) {
       String partOfSpeech = token.getPartOfSpeech().getTag().toString();
-      if(partOfSpeech.equals("ADJ")) {
-        adjectives.add(token.getLemma().toLowerCase());
+      
+      if(partOfSpeech.equals("VERB") && !add) {
```

Comment:
https://google.github.io/styleguide/javaguide.html#s4.6.2-horizontal-whitespace for if statement